internal/service/federatedsettingsidentityprovider/data_source_federated_settings_identity_provider.go

```diff
@@ -281,11 +281,7 @@ func dataSourceMongoDBAtlasFederatedSettingsIdentityProviderRead(ctx context.Con
 		return diag.FromErr(fmt.Errorf("error setting `status` for federatedSettings IdentityProviders: %s", err))
 	}
 
-	if err := d.Set("id", federatedSettingsIdentityProvider.Id); err != nil {
-		return diag.FromErr(fmt.Errorf("error setting `id` for federatedSettings IdentityProviders: %s", err))
-	}
-
-	d.SetId(federatedSettingsIdentityProvider.OktaIdpId)
+	d.SetId(federatedSettingsIdentityProvider.Id)
```

Comment:
is the PR title correct? i was expecting only doc changes

Reply:
I updated the title, sorry about that. 
The changes are minor things which I was unable to test before SDK was updated and now I fixed them